decentraland/unity-explorer
# Fix: released resources related to splash screen logo animation

## What does this PR change?

Fix #2119 
This PR releases the memory used by the sprites of the animation of the DCL logo

...

## How to test the changes?



1. Launch the explorer
2. Verify that the dcl logo in the initial loading is still playing its animation

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
